vms/platformvm/txs/fee/complexity.go

```diff
@@ -182,9 +178,6 @@ var (
 			wrappers.IntLen + // number of inputs
 			wrappers.IntLen + // length of memo
 			wrappers.IntLen, // number of credentials
-		gas.DBRead:  0,
-		gas.DBWrite: 0,
-		gas.Compute: 0,
 	}
 	IntrinsicConvertSubnetTxComplexities = gas.Dimensions{
```

Comment:
this might seem like an overkill and redundant, so i'm not sure it's worth investing the time, but - would it not be possible to annotate the functions in `standard_tx_executor.go` with "W" and "R" to have better book-keeping of this file and where we pay for each R/W operation?

Reply:
nevermind, since some of the writes are being performed via `state.write()` i think it's not that feasible.